demos/logging/src/main/java/example/app/FileErrorManager.java

```diff
@@ -1,5 +1,6 @@
+package example.app;
 /*
- * Copyright (c) 2009, 2019 Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2009, 2022 Oracle and/or its affiliates. All rights reserved.
  * Copyright (c) 2009, 2019 Jason Mehrens. All Rights Reserved.
  *
  * This program and the accompanying materials are made available under the
```

Comment:
FileErrorManager::wrap needs to update the reflective classname for NewlineOutputStream since that is now in a package.

Reply:
Nice catch, thank you.